sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_client.py

```diff
@@ -217,6 +217,18 @@ def list_submitted_jobs(self, **kwargs):
         # type: (**Any) -> ItemPaged[JobStatusResult]
         """List all the submitted translation jobs under the Document Translation resource.
 
+        :keyword int top: indicates the total number of records the user wants to be returned across all pages.
+        :keyword int skip: indicates the number of records to skip from the list of batches based on the sorting
+            method specified. By default, we sort by descending start time.
+        :keyword int maxpagesize: is the maximum items returned in a page.  If more items are requested via top (or
+            top is not specified and there are more items to be returned), @nextLink will contain the link
+            to the next page.
+        :keyword list[str] ids: Ids to use in filtering.
```

Comment:
job or document ID? I think we should rename this to be specific that we are filtering by job_id here.